server/routerlicious/packages/lambdas/src/utils/messageGenerator.ts

```diff
@@ -11,13 +11,19 @@ import {
     NackErrorType,
 } from "@fluidframework/protocol-definitions";
 
-export const createNackMessage = (code: number, type: NackErrorType, message: string): INack => ({
+export const createNackMessage = (
+    code: number,
+    type: NackErrorType,
+    message: string,
+    retryAfterInSec?: number,
+): INack => ({
     operation: undefined,
     sequenceNumber: -1,
     content: {
         code,
         type,
         message,
+        retryAfter: retryAfterInSec,
```

Comment:
can we update this to retryAfterInSec too?

Reply:
Updating INackContent.retryAfter to retryAfterInSec has a pretty massive scope. IMO it should be a separate item with a few PRs